schema/workflow/base_flowchart.json

```diff
@@ -0,0 +1,47 @@
+{
+    "schemaId": "workflow-base-flowchart",
```

Comment:
Should this be the `BaseSubworkflowSchema` as defined in `/imports/workflows/schemas`? If so, then the fields `isMultiMaterial` and `systemName` still need to be added.

Reply:
Let's call this "base_flow" just to avoid confusion and be logically (and literally) incorporated in workflow and subworkflow